### ISSUE TITLE
Adding Minimum Quantity for New-SnipeITComponent

### ISSUE THREAD

**Issue by Christopheric1** [snazy2000/SnipeitPS#168]
I would like to add -min_qty to the New-SnipeItComponent function. Currently, it has the -qty, but I would also like to set the minimum from the Powershell API if possible. 

Thank you!

**Comment by PetriAsi** [comment]
parameter  will be named -min_amt as we are using same field names as Snipe It api.

**Comment by PetriAsi** [comment]
Fixed in v1.7.159

**Comment by Christopheric1** [comment]
I just tried the following command:

```
New-SnipeITComponent -Name $Monitor -category_id 5 -qty $MonQty -company_id 1 -min_amt 2
```

and got the following error:

```
New-SnipeitComponent : A parameter cannot be found that matches parameter name 'min_amt'. This error might have been caused by applying the default parameter binding. You can disable the default parameter binding in $PSDefaultParameterValues by setting 
$PSDefaultParameterValues["Disabled"] to be $true, and then trying again. The following default parameters were successfully bound for this cmdlet when the error occurred: -url -apiKey
At C:\Snipe-IT-Add-Item.ps1:986 char:11
+           New-SnipeITComponent -Name $Monitor -category_id 5 -qty $Mo ...
+           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [New-SnipeitComponent], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,New-SnipeitComponent
```

I have updated to the latest version as well
```
Script     1.7.159    SnipeitPS                           {Get-SnipeitAccessory, Get-SnipeitAccessoryOwner, Get-SnipeitActivity, Get-SnipeitAsset...} 
```
